### DISCUSSION TITLE
How do I use this?

### DISCUSSION THREAD

**Discussion by jiriks74** [CalebQ42/LinuxPA#15] [Q&A]
Hi, 
I'd like to try out your app, but I have no idea how to use this project. Would you help me out, please?

**Comment by CalebQ42** [comment]
What exactly do you need help with? The general idea is the LinuxPA executable is supposed to be put at the root of your flashdrive (or wherever you want to use LinuxPA), and then the apps you want to watch inside a folder named PortableApps. You can place AppImages directly into the PortableApps folder, but it's better if you put your programs first into a separate folder. A folder structure would look something like this:

```
Flash Drive
|- LinuxPA
|- PortableApps
  |- Firefox
    |- Firefox.AppImage
```

Additionally, make sure the LinuxPA executable has exec permission. If your trying to run it on a FAT32 flash drive, make sure you mount it with the necessary options to allow executing (I have the mount options in the project's README).

**Reply by jiriks74** [reply]
> What exactly do you need help with? The general idea is the LinuxPA executable is supposed to be put at the root of your flashdrive (or wherever you want to use LinuxPA), and then the apps you want to watch inside a folder named PortableApps. You can place AppImages directly into the PortableApps folder, but it's better if you put your programs first into a separate folder. A folder structure would look something like this:
> 
> ```
> Flash Drive
> |- LinuxPA
> |- PortableApps
>   |- Firefox
>     |- Firefox.AppImage
> ```
> 
> Additionally, make sure the LinuxPA executable has exec permission. If your trying to run it on a FAT32 flash drive, make sure you mount it with the necessary options to allow executing (I have the mount options in the project's README).

I mean how do I compile this project? I haven't been able to figure out how to get the LinuxPA executable yet.

**Reply by CalebQ42** [reply]
Well you can either download the LinuxPA executable on the [latest release](https://github.com/CalebQ42/LinuxPA/releases). If you want to compile it, make sure you have Go installed (go.dev) and simply run `go install`.

**Reply by jiriks74** [reply]
Thanks, that's what I needed. I've never used go before, so I have no idea how it works. I'll try it when I get home.
